vnet/lib/vnet/endpoints/1.0/datapaths.rb

```diff
@@ -41,16 +41,21 @@ def self.put_post_shared_params
   end
 
   param_uuid M::Interface, :interface_uuid, required: true
-  param :broadcast_mac_address, :String, required: true, transform: PARSE_MAC
+
+  # 'broadcast_mac_address' is deprecated and should be phased out.
+  param :mac_address, :String, transform: PARSE_MAC
```

Comment:
These aren't the models. These are the WebAPI endpoints.

I double checked with @akry to make sure we are on the same page as to what `mac_address` does for us in a datapath_network. If I understand correctly, the MAC2MAC protocol won't be able to do ARP broadcast without it. Therefore it should be a required API parameter until we create a policy to generate them when not provided.

Am I wrong?